checker/src/main/java/org/checkerframework/checker/signedness/SignednessUtil.java

```diff
@@ -358,11 +364,11 @@ public static void getUnsigned(ByteBuffer b, @Unsigned byte[] bs) {
     /**
      * Compares two unsigned longs x and y.
      *
-     * <p>This is a reimplementation of Java 8's {@code Long.compareUnsigned(long, long)}.
-     *
      * @return a negative number iff x {@literal <} y, a positive number iff x {@literal >} y, and
      *     zero iff x == y.
+     * @deprecated use Java 8's {@code Long.compareUnsigned(long, long)}.
```

Comment:
Elsewhere you used `@link`, which is useful in the Javadoc.  Why did you use `@code` here?

Reply:
It was written earlier like this in the doc. Should I change it to ```@link```?

Reply:
I see.  The previous writing isn't your fault, but it would still be nice to make it a link.
Thanks!